salt/_modules/metalk8s_certs.py

```diff
@@ -0,0 +1,34 @@
+'''
+Various functions for certs generation.
+'''
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'metalk8s_certs'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def compute_subject_alt_name(subject_alt_name):
+    '''
+    Use a dict to compute the suject alternative name string for a certificate
+
+    >>> compute_subject_alt_name({'DNS': ['kubernetes', 'kubernetes.default']})
+    'DNS:kubernetes, DNS:kubernetes.default'
+    >>> compute_subject_alt_name({'DNS': ['kubernetes', 'kubernetes.default'], 'IP': ['1.2.3.4', '5.6.7.8']})
+    'IP:1.2.3.4, IP:5.6.7.8, DNS:kubernetes, DNS:kubernetes.default'
```

Comment:
We probably want to add a mention that output order is meaningless since it's not assured because of the `.items()` call.